instagram_clone/spec/features/image_feature_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'rails_helper'
+require 'helpers'
+
+RSpec.configure do |c|
+  c.include Helpers
+end
+
+feature 'images' do
+
+  context 'no images have been added' do
+    scenario 'should display no images' do
+      visit '/images'
+      expect(page).to have_content 'No images yet'
+    end
+  end
+
+  context 'posting images' do
+    scenario 'user can see image upload link once logged in' do
+      sign_up
+      expect(page).to have_link 'Upload an image'
+    end
+
+    scenario 'adds the image to the homepage' do
+      sign_up
+      upload_image
+      expect(page).to have_content 'Cake'
+    end
+  end
+
+  context 'viewing images' do
+
+    let!(:otter){Image.create(name:'otter')}
```

Comment:
Space missing after colon.<br>Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.